Wireframe/index.html

```diff
@@ -1,13 +1,112 @@
+  <link rel="shortcut icon" type="image/x-icon" href="img/github-logo-5F384D0265-seeklogo.com.png" />
+  <script src="https://kit.fontawesome.com/c0b874b18b.js" crossorigin="anonymous"></script>
+  <title>Git WireFrame</title>
+  <link rel="stylesheet" href="/Wireframe/css/style.css">
+</head>
+
+<body>
+  <header>
+    <h1>GIT Control Versions</h1>
+    <p>
+      Git is a free and open source distributed version control system designed to handle everything from small to very
+      large projects with speed and efficiency.
+    </p>
+  </header>
+  <main>
+    <section class="hero">
+      <div>
+        <img src="/Wireframe/img/giff.gif" alt="Giff showing how works the git control version" class="img-hero">
+      </div>
+      <h2>What is Git</h2>
+      <p class="overlay">
+        By far, the most widely used modern version control system in the world today is Git. Git is a mature, actively
+        maintained open source project originally developed in 2005 by Linus Torvalds, the famous creator of the Linux
+        operating system kernel.<br><br>
+
+        A staggering number of software projects rely on Git for version control, including commercial projects as well
+        as open source. Developers who have worked with Git are well represented in the pool of available software
+        development talent and it works well on a wide range of operating systems and IDEs (Integrated Development
+        Environments).<br><br>
+
+        Having a distributed architecture, Git is an example of a DVCS (hence Distributed Version Control System).
+        Rather than have only one single place for the full version history of the software as is common in once-popular
+        version control systems like CVS or Subversion (also known as SVN), in Git, every developer's working copy of
+        the code is also a repository that can contain the full history of all changes.
+      </p>
+      <button onclick="window.open('https://git-scm.com/book/en/v2/Getting-Started-What-is-Git%3F', '_blank')">Read
+        More</button>
+    </section>
+    <section class="wrapper">
+      <article class="article-wrapper">
+        <div>
+          <img src="/Wireframe/img/git.jpg" alt="image of git" class="img-article">
+        </div>
+        <h2>Why do developers need Git?</h2>
+        <p class="overlay">
+          Developers need Git because it is a vital tool for version control that facilitates efficient collaboration,
+          organization, and management of code. As software projects become more complex, keeping track of changes and
+          coordinating work among multiple team members becomes challenging. Git addresses these issues by allowing
+          developers to create branches for their work, enabling them to make changes independently without affecting
+          the main codebase. This branching strategy helps avoid conflicts and allows for experimentation with new
+          features.<br><br><br>
```

Comment:
If it's a new paragraph, make it a new paragraph, please! Don't use breaks to make paragraphs.